core/src/main/java/com/linecorp/armeria/client/ClientFactoryProvider.java

```diff
@@ -17,10 +17,12 @@
 package com.linecorp.armeria.client;
 
 import com.linecorp.armeria.common.SerializationFormat;
+import com.linecorp.armeria.common.util.UnstableApi;
 
 /**
  * Creates a new {@link ClientFactory} dynamically via Java SPI (Service Provider Interface).
  */
+@UnstableApi
```

Comment:
Does this effectively mean we only officially support our official client implementations, and user-created custom ones may potentially have breaking changes? I'm ok with either but we may want to document that explicitly (I lean towards not marking this unstable though it seems pretty well-baked).

Reply:
I'm somewhat unsure about the stability of this API although it stayed unchanged for a very long time. For instance, a user may want to build a `ClientFactory` which is built on top of `GrpcClientFactory` or `THttpClientFactory`.